parm/config/config.resources

```diff
@@ -71,8 +72,8 @@ elif [ $step = "wavepostsbs" ]; then
 
 elif [ $step = "wavepostbndpnt" ]; then
 
-    export wtime_wavepostbndpnt="02:00:00"
-    export npe_wavepostbndpnt=280
+    export wtime_wavepostbndpnt="01:00:00"
```

Comment:
We do not use this job at all for s2s yet, so why is this being changed here?

Reply:
This still needs to be reverted to what's in develop.

Reply:
Will revert, thanks!